packages/datatrak-web/src/layout/UserMenu/MenuList.tsx

```diff
@@ -46,69 +56,57 @@ export const MenuList = ({
   children?: ReactNode;
   onCloseMenu: () => void;
 }) => {
-  const { isLoggedIn } = useCurrentUser();
+  const { isLoggedIn, projectId } = useCurrentUser();
   const { mutate: logout } = useLogout();
 
-  // The help centre link is the same for both logged in and logged out users
-  const helpCentre = {
+  const accountSettingsItem = {
+    label: 'Account settings',
+    to: ROUTES.ACCOUNT_SETTINGS,
+  };
+  // The help centre link is the same for both logged-in and logged-out users
+  const helpCentreItem = {
     label: 'Help centre',
     href: 'https://beyond-essential.slab.com/posts/tupaia-instruction-manuals-05nke1dm',
     isExternal: true,
     component: Link,
   };
-  const onClickLogout = () => {
-    logout();
-    onCloseMenu();
+  const logOutItem = {
+    label: 'Log out',
+    onClick: () => {
+      logout();
+      onCloseMenu();
+    },
+    component: 'button',
   };
 
-  const menuItems = isLoggedIn
-    ? [
-        {
-          label: 'Account settings',
-          to: ROUTES.ACCOUNT_SETTINGS,
-        },
-        helpCentre,
-        {
-          label: 'Log out',
-          onClick: onClickLogout,
-          component: 'button',
-        },
-      ]
-    : [helpCentre];
+  const hasProjectSelected = !!projectId;
+
+  const menuItems: MenuItem[] = [];
+  if (isLoggedIn && hasProjectSelected) {
+    menuItems.push(accountSettingsItem);
+  }
+  menuItems.push(helpCentreItem);
+  if (isLoggedIn) {
+    menuItems.push(logOutItem);
+  }
```

Comment:
This here is the bulk of the change. The other changed lines are just shifting around existing code to more tidily facilitate the new logic here